src/bioregistry/data/bioregistry.json

```diff
@@ -72592,6 +72592,30 @@
         "pmc": "PMC2475545",
         "pubmed": "18651794",
         "title": "WikiPathways: pathway editing for the people."
+      },
+      {
+        "doi": "10.1093/NAR/GKAA1024",
```

Comment:
seems like these are case insensitive to the DOI resolver but the article page shows it in lowercase so we should probably stick to that

Reply:
I learned from the Wikidata project that DOIs are by design case insensitive

Reply:
Interesting, so if it's a free-for-all, then we'll require they're standardized to lowercase in the Bioregistry.

Reply:
I can live with that. I'll make an update commit.